src/app/_components/DesktopNavigation.tsx

```diff
@@ -70,13 +70,18 @@ function SubNavItem({
   )
 }
 
+const mainNavItem = {
+  width: 'px-4',
+  offsetWidth: '-mr-4',
+}
+
```

Comment:
I think we should try to standardize these. Using `touchTarget` everywhere.

```
const touchTarget = {
  class: 'px-4',
  offsetClass: '-mr-4',
}
```

Happy to suffix with the element name, if you think it's clearer: `touchTargetMainNavItem`